bootstrap.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+echo 'root access'
+apt-get update
+echo 'update'
+apt-get install git -y
+echo 'git install'
+apt-get install openjdk-18-jre-headless -y
+echo 'java install'
+apt-get install maven -y
+echo 'maven install'
+apt-get install graphviz -y
+echo 'graphviz install'
+cd ~/Desktop
+git clone https://github.com/bitslab/java-callgraph
```

Comment:
This line require github permissions, and will fail.  Instead, clone from the disk.  Note that the repo should be mounted somewhere inside the VM.  So use that path instead.

Reply:
changed git clone to cp from /vagrant